### PR TITLE
Consolidate subtitle dialogs

### PR DESCRIPTION
**Pull request by tamland**
Currently we have two subtitle dialogs, one part of audio settings and one custom skin menu, both of which are missing some functions. So this merge them into one where you can access everything from one place.

### CODE REVIEW

File: addons/skin.estuary/xml/Custom_1101_SettingsList.xml [hunk 1]
```diff
@@ -68,50 +74,6 @@
...
-						<include>DialogSettingButton</include>
-						<label>$LOCALIZE[22006]</label>
-						<label2>$INFO[Player.SubtitleDelay,[B],[/B]]</label2>
-						<onclick>SubtitleDelay</onclick>
-						<enable>VideoPlayer.HasSubtitles + VideoPlayer.SubtitlesEnabled</enable>
-					</control>
-					<control type="button" id="12103">
-						<width>700</width>
-						<include>DialogSettingButton</include>
-						<label>$LOCALIZE[31051]</label>
```
Comment (phil65) [inline addons/skin.estuary/xml/Custom_1101_SettingsList.xml]: This label can probably also get removed from skin .po file?

**Review by phil65 — COMMENTED** [review]
Didnt test, but sounds good and makes sense, +1.
A quick screenshot of that new dialog would be nice.
Skin changes look fine.
Thx!

File: addons/skin.estuary/xml/Custom_1101_SettingsList.xml (hunk 1 above)
Comment (tamland) [reply addons/skin.estuary/xml/Custom_1101_SettingsList.xml]: done

**Comment by tamland** [comment]
> Didnt test, but sounds good and makes sense, +1.
A quick screenshot of that new dialog would be nice.
Skin changes look fine.
Thx!

It looks the same as the video and the audio dialogs.

**Pull request merged**